doc/api/readline.md

```diff
@@ -722,6 +723,123 @@ const { createInterface } = require('readline');
 })();
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
```

Comment:
It's probably more intuitive to list the not supported platforms as note next to the individual entry instead of below.

```suggestion
    <th>Description</th>
    <th>Notes</th>
```

Reply:
@BridgeAR, thanks. Fixed it.